systemtest/src/main/java/io/strimzi/systemtest/resources/ResourceOperation.java

```diff
@@ -42,7 +46,7 @@ public static long getTimeoutForResourceReadiness(String kind) {
                 timeout = Duration.ofMinutes(4).toMillis();
                 break;
             default:
-                timeout = Duration.ofMinutes(2).toMillis();
+                timeout = Duration.ofMinutes(3).toMillis();
```

Comment:
Why this change ? :) 2 minutes were not enough?

Reply:
I saw that in some cases it wasn't enough, by default it was set to 3 minutes, so I just changed it back.